test/functional/feature_segwit.py

```diff
@@ -154,8 +154,8 @@ def run_test(self):
         self.skip_mine(self.nodes[2], p2sh_ids[NODE_2][WIT_V1][0], True)  # block 427
 
         self.log.info("Verify unsigned p2sh witness txs without a redeem script are invalid")
-        self.fail_accept(self.nodes[2], "mandatory-script-verify-flag", p2sh_ids[NODE_2][WIT_V0][1], False)
-        self.fail_accept(self.nodes[2], "mandatory-script-verify-flag", p2sh_ids[NODE_2][WIT_V1][1], False)
+        self.fail_accept(self.nodes[2], "script-verify-flag", p2sh_ids[NODE_2][WIT_V0][1], False)
```

Comment:
The failure string can be "script-verify-flag-failed" here. Even better, with the specific error string "script-verify-flag-failed (Operation not valid with the current stack size)"

Reply:
Leaving this one alone -- someone interested in improving this test can take it on in a separate PR.